samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Shell.xaml

```diff
@@ -8,9 +8,11 @@
 			 xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 			 xmlns:wasm="http://uno.ui/wasm"
 			 xmlns:xamarin="http://uno.ui/xamarin"
+			 xmlns:utu="using:Uno.Toolkit.UI"
 			 mc:Ignorable="wasm xamarin">
 
-	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+	<Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+		  utu:SafeArea.Insets="Top, Bottom">
```

Comment:
We might as well just set this to `"VisibleBounds"` instead

Reply:
Do you mean, use VisibleBoundsPadding.PaddingMask instead of SafeArea.Insets?

Reply:
No instead of `utu:SafeArea.Insets="Top, Bottom"` you can put `utu:SafeArea.Insets="VisibleBounds"`